README.md

```diff
@@ -172,6 +172,14 @@ export XHARNESS_MLAUNCH_PATH='[xharness root]/artifacts/obj/Microsoft.DotNet.XHa
 ```
 and you won't have to specify the `--mlaunch` argument.
 
+### Running E2E tests
+In case you want to test some of the XHarness flows in Helix, you can run E2E tests located in `/tests/integration-tests`. These usually download some pre-built application and send it to our "test cloud" called Helix together with an XHarness version built from your sources. There, XHarness executes the app on a device/simulator and report back. This way you can see your changes apply in an E2E test.
```

Comment:
```suggestion
In case you want to test your changes in XHarness, you can run E2E tests located in `/tests/integration-tests`. These usually download some pre-built application and send it to our "test cloud" called Helix together with an XHarness version built from your sources. There, XHarness executes the app on a device/simulator.
```